examples/stable_diffusion_xl/demo/sampling.py

```diff
@@ -3,6 +3,9 @@
 import os
 import time
 
+if os.environ.get("MS_PYNATIVE_GE") != "1":
```

Comment:
Should check MS version before setting environment variable in code

Reply:
It is harmless on the current device